templates/partials/contents.html

```diff
@@ -1,14 +1,26 @@
+{% set definition_count = namespace(value=0) %}
+
 {% for item in contents %}
+  {%- if item.definition -%}
+    {% set definition = item.definition %}
+    {% set definition_id = 'definition-' ~ definition_count.value %}
```

Comment:
Could `loop.index` be used here like it is in `question-definition.html`?

Reply:
Problem is I have other elements in there as well not just definition, so loop.index would look odd and would be hard to test.

Reply:
Ah, I see. The value should start at 1 to be consistent with `loop.index` (which starts at 1).